src/style.css

```diff
@@ -0,0 +1,186 @@
+/* start index styles */
+
+/*
+1.index styles.
+2.universal styles.
+3.common styles.
+4.content styles.
+5.media queries styles.
+*/
+
+/* end index styles */
+
+/* start universal styles */
```

Comment:
I think it would be better if you have not included those in your code, since they are not necessary and doesn't effect the actual website